### ISSUE TITLE
sound and lamp LED offset?

### ISSUE THREAD

**Issue by AARP41298** [psieg/Lightpack#468]
Now i understand why the youtube tutorials start to tape the LED strip from the bottom center instead of one corner
The offset of the configuration wizard only works with screen grabbing not with the other modes

thankfully i dont tape my LED strip yet

**Comment by psieg** [comment]
Right, the offset that's set in the wizard only applies to screen grabbing. For Mood lamp the order does not matter but Sound Visualization also isn't aware of it.
